src/main/java/winston/AddDeadlineCommand.java

```diff
@@ -0,0 +1,24 @@
+package winston;
+
+/**
+ *  Represents the deadline command from the parent abstract class Command.
```

Comment:
This is a really minor suggestion: maybe the extra space at the start of the description block can be removed for consistency. This is present for a few of the command classes.
```suggestion
 * Represents the deadline command from the parent abstract class Command.
```